ValetTouchIDTest/ValetTouchIDTestViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ValetTouchIDTestViewController.swift
+//  Valet
+//
+//  Created by Eric Muller on 4/20/16.
+//  Copyright © 2016 Square, Inc. All rights reserved.
+//
+
+import UIKit
+
+
+final class ValetTouchIDTestViewController : UIViewController
+{
+    // MARK: Properties
+
+    @IBOutlet var textView : UITextView?
+    let secureEnclaveValet : VALSecureEnclaveValet = VALSecureEnclaveValet.init(identifier: "UserPresence", accessControl: VALAccessControl.UserPresence) as VALSecureEnclaveValet!
```

Comment:
nit: no need to specify type on the left hand side.
Also, why on earth do you need to `as VALSecureEnclaveValet!`? That really shouldn't be necessary.

The following should do:
`let secureEnclaveValet = VALSecureEnclaveValet.init(identifier: "UserPresence", accessControl: VALAccessControl.UserPresence)`
 
If it doesn't... we should fix our initializers to work better with Swift.